stripe/src/main/java/com/stripe/android/view/NetbankingBank.kt

```diff
@@ -0,0 +1,452 @@
+        "tnc",
+        "Tamil Nadu State Co-operative Bank"
+    ),
+    UBI(
+        "UBI",
+        "ubi",
+        "Union Bank of India"
+    ),
+    UNIBank(
+        "UNI",
+        "united_bank_of_india",
+        "PNB (Erstwhile-United Bank of India )"
+    ),
+    AxisBank(
+        "UTI",
+        "axis",
+        "Axis Bank"
+    ),
+    VijayaBank(
+        "VJB",
+        "vijaya",
+        "Vijaya Bank"
+    ),
+    VarachhaBank(
+        "VRB",
+        "varachha",
+        "Varachha Co-operative Bank Limited"
+    ),
+    YesBank(
+        "YBK",
+        "yes",
+        "Yes Bank"
+    ),
+    ZoroastrianBank(
+        "ZOB",
+        "zoroastrian",
+        "Zoroastrian Co-operative Bank Limited"
+    ),
+    KarnatakaGraminBank(
+        "PKB",
+        "karnataka_gramin",
+        "Karnataka Gramin Bank"
+    ),
+    ShamraoVithalBank(
+        "SVC",
+        "shamrao_vithal",
+        "Shamrao Vithal Co-op Bank"
+    ),
+    NutanNagrikBank(
+        "NUT",
+        "nutan_nagrik",
+        "Nutan Nagrik Bank"
+    ),
+    BOBBank(
+        "BBC",
+        "bob_corp",
+        "Bank of Baroda - Corporate Banking"
+    ),
+    PunjabNationalCorpBank(
+        "CPN",
+        "pnb_corp",
+        "Punjab National Bank - Corporate Banking"
+    ),
+    ShamraoVithalCorpBank(
+        "SV2",
+        "shamrao_vithal_corp",
+        "Shamrao Vithal Co-op Bank - Corporate"
+    ),
+    BNPParibas(
+        "BNP",
+        "bnp_paribas",
+        "BNP Paribas"
+    ),
+    RBLCorpBank(
+        "RTC",
+        "rbl_corp",
+        "RBL Bank Limited - Corporate Banking"
+    ),
+    ICICICorpBank(
+        "ICO",
+        "icici_corp",
+        "ICICI Corporate Netbanking"
+    ),
+    IDBICorpBank(
+        "IDC",
+        "idbi_corp",
+        "IDBI Corporate"
+    ),
+    AxisCorpBank(
+        "AXC",
+        "axis_corp",
+        "Axis Bank Corporate"
+    ),
+    AndhraCorpBank(
+        "ADC",
+        "andhra_corp",
+        "Andhra Bank Corporate"
+    ),
+    DhanlaxmiCorpBank(
+        "DL2",
+        "dhanlaxmi_corp",
+        "Dhanlaxmi Bank Corporate"
+    ),
+    AllahabadCorpBank(
+        "ALC",
+        "allahabad_corp",
+        "Allahabad Bank Corporate"
+    ),
+    HFDCCorpBank(
+        "CH3",
+        "hdfc_corp",
+        "HDFC Bank Corporate"
+    ),
+    YesCorpBank(
+        "YBC",
+        "yes_corp",
+        "Yes Bank Corporate"
+    ),
+    CorporationBankCorporate(
+        "CR2",
+        "corporation_bank_corp",
+        "Corporation Bank - Corporate"
+    ),
+    BarclaysCorpBank(
+        "BRL",
+        "barclays_corp",
+        "Barclays Bank - Corporate Net Banking"
+    );
+
+    companion object {
+        /**
+         * Return the [NetbankingBank] that matches the given bank code (e.g. "bank_of_india"),
+         * or null if no match is found.
+         *
+         * The bank code should be obtained from [PaymentMethod.NetbankingBank.bank].
+         */
+        @JvmStatic
```

Comment:
We can probably remove `@JvmStatic` - are we using it somewhere?

Reply:
it is used in the example Netbanking activity:
`final NetbankingBank netbankingBank = NetbankingBank.get(netbankingCode);`